sfa_dash/blueprints/main.py

```diff
@@ -151,31 +288,27 @@ def get(self, uuid, **kwargs):
             self.temp_args.update({'errors': e.errors})
         else:
             try:
-                self.metadata['site'] = self.get_site_metadata(
-                    self.metadata['site_id'])
+                self.set_site_or_aggregate_metadata()
             except DataRequestException:
-                self.temp_args.update({
-                    'warnings': {
-                        'Site Access': [
-                            'Site inaccessible. Plots will not be displayed.']
-                    },
-                    'plot': None,
-                })
+                self.temp_args.update({'plot': None})
             else:
                 # If site data is available, try to create a plot
-                self.insert_plot(uuid, start, end)
+                if self.metadata.get('site') is not None:
```

Comment:
should not have to check this once ``blueprints/util.py`` can handle either site or agg